core/domain/opportunity_services.py

```diff
@@ -362,6 +362,29 @@ def get_voiceover_opportunities(language_code, cursor):
     return opportunities, cursor, more
 
 
+def get_opportunities_by_ids(ids):
```

Comment:
The naming of this function suggests "opportunities", but the body of the function suggests "opportunity summaries". Please standardize the naming and ensure that it is consistent.